guides/calculators/tax-calculator.md

```diff
@@ -0,0 +1,24 @@
+# Tax calculator
+
+Solidus comes with a tax calculator that is used to calculate both sales tax
+(United States-style taxes) and value-added tax (VAT):
+[`Spree::Calculator::DefaultTax`][default-tax-calculator]. Typically, this
+calculator should be the only tax calculator required by your store.
+
+Taxes can apply to line items, shipments, or an entire order.
+
+The tax calculator uses its calculable – a `Spree::TaxRate` – to calculate tax
+totals. Tax rates are represented as decimal between `0.01` and `1.00`. 
```

Comment:
I don't believe this is true. A tax rate has a precision of 8, scale 5 at the DB level and no such restrictions in rails validations that I know of.